modules/filecheck/charts.go

```diff
@@ -0,0 +1,63 @@
+package filecheck
+
+import "github.com/netdata/go-orchestrator/module"
+
+var (
+	fileCharts = module.Charts{
+		fileExistenceChart.Copy(),
+		fileModTimeAgoChart.Copy(),
+		fileSizeChart.Copy(),
+	}
+
+	fileExistenceChart = module.Chart{
+		ID:    "file_existence",
+		Title: "File Existence (0: doesnt exist, 1: exists)",
+		Units: "boolean",
+		Fam:   "files",
+		Ctx:   "filecheck.file_existence",
+	}
+	fileModTimeAgoChart = module.Chart{
+		ID:    "file_mtime_ago",
+		Title: "File Time Since the Last Modification",
+		Units: "seconds",
+		Fam:   "files",
+		Ctx:   "filecheck.file_mtime_ago",
+	}
+	fileSizeChart = module.Chart{
+		ID:    "file_size",
+		Title: "File Size",
+		Units: "bytes",
+		Fam:   "files",
+		Ctx:   "filecheck.file_size",
+	}
+)
+
+var (
+	dirCharts = module.Charts{
+		dirExistenceChart.Copy(),
+		dirModTimeChart.Copy(),
+		dirNumOfFilesChart.Copy(),
+	}
+
+	dirExistenceChart = module.Chart{
+		ID:    "dir_existence",
+		Title: "Dir Existence (0: doesnt exist, 1: exists)",
```

Comment:
It is also necessary to adjust the title here, `doesn't` instead `doesnt`.

Reply:
see above
i am pretty sure plugins.d parser can't handle `'` inside `'`

We send it as 
```
CHART 'filecheck_netdata_log.file_existence' '' 'File Existence (0: doesnt exist, 1: exists)' 'boolean' 'files' 'filecheck.file_existence' '' '70000' '10' '' 'go.d' 'filecheck'
```